jakarta-package-alignment/src/test/groovy/com/palantir/gradle/jakartapackagealignment/JakartaPackageAlignmentPluginIntegrationSpec.groovy

```diff
@@ -16,13 +16,99 @@
 
 package com.palantir.gradle.jakartapackagealignment
 
+import groovy.transform.CompileStatic
 import nebula.test.IntegrationSpec
+import nebula.test.dependencies.DependencyGraph
+import nebula.test.dependencies.GradleDependencyGenerator
 
 /**
  * Integration tests for {@link JakartaPackageAlignmentPlugin}.
  */
 class JakartaPackageAlignmentPluginIntegrationSpec extends IntegrationSpec {
 
-    // TODO
+    static def PLUGIN_NAME = "com.palantir.jakarta-package-alignment"
+    private File mavenRepo
+    private File versionsProps
 
+    // borrowed from gcv
+    @CompileStatic
```

Comment:
```suggestion
```

iirc `CompileStatic` is a performance optimization which interacts poorly with a few things. Let's not use it unless we find we really need to

Reply:
👍 fixed